react-main/src/redux/logout/actions.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { push } from 'react-router-redux';
+
+export const actions = {
+  LOGOUT: 'LOGOUT'
+};
+
+const actionCreatorsLogout = {
+  logout: dispatch => (dispatch(push('/')), {type: actions.LOGOUT, token: ''})
```

Comment:
this is unnecessary. You can just call `dispatch(push('/'))`from the mapDispatchToProps of your component